pkg/controller/config/config_controller.go

```diff
@@ -65,20 +67,41 @@ func newReconciler(mgr manager.Manager) (reconcile.Reconciler, error) {
 		return nil, err
 	}
 
+	nonRedHat, err := fetchNonRedHat(mgr)
+	if err != nil {
+		log.Error(err, "error fetching community resources")
+		nonRedHat = mf.Manifest{}
+	}
+
 	secClient, err := sec.NewForConfig(mgr.GetConfig())
 	if err != nil {
 		return nil, err
 	}
 
 	return &ReconcileConfig{
-		client:    mgr.GetClient(),
-		scheme:    mgr.GetScheme(),
-		secClient: secClient,
-		pipeline:  pipeline,
-		addons:    addons,
+		client:             mgr.GetClient(),
+		scheme:             mgr.GetScheme(),
+		secClient:          secClient,
+		pipeline:           pipeline,
+		addons:             addons,
+		nonRedHatResources: nonRedHat,
 	}, nil
 }
 
+func fetchNonRedHat(mgr manager.Manager) (mf.Manifest, error) {
+	if flag.SkipNonRedHatResources {
+		return mf.Manifest{}, nil
+	}
+	//manifestival can take urls/filepaths as input
```

Comment:
did you apply go fmt?

Reply:
yes, it is configured to run automatically.
This looks like github counting tabs differently as we don't have a `.editorconfig` file.